### DISCUSSION TITLE
how to change the background color of legend area

### DISCUSSION THREAD

**Discussion by saunter999** [highfestiva/finplot#464] [Q&A]
Could some one tell me how to change the background color of legend area? Thanks
<img width="158" alt="1701002118026_19B132F0-053A-4ee0-93B6-D6F8037F2671" src="https://github.com/highfestiva/finplot/assets/44839827/342d927f-3c43-4732-8f5e-e2216d94d70a">

**Comment by highfestiva** [comment]
Try:
```python
import finplot
finplot.legend_fill_color = '#202020d0'
...
```
